lang/nl/strings.php

```diff
@@ -58,21 +76,25 @@
     'action' => 'Actie',
     'data' => 'Gegevens',
     'queued' => 'In wachtrij',
-    'last_run' => 'Laatst uitgevoerd',
-    'next_run' => 'Volgende uitvoering',
-    'not_run_yet' => 'Nog niet uitgevoerd',
+    'last_run' => 'Laatst Uitgevoerd',
+    'next_run' => 'Volgende Uitvoering',
+    'not_run_yet' => 'Nog Niet Uitgevoerd',
     'yes' => 'Ja',
     'no' => 'Nee',
     'delete' => 'Verwijderen',
-    '2fa' => 'Tweestapsverificatie',
+    '2fa' => '2FA',
     'logout' => 'Uitloggen',
     'admin_cp' => 'Admin Controle Paneel',
     'optional' => 'Optioneel',
-    'read_only' => 'Alleen lezen',
+    'read_only' => 'Alleen Lezen',
+    'read_only?' => 'Alleen Lezen?',
+    'writable' => 'Schrijfbaar',
     'relation' => 'Relatie',
     'owner' => 'Eigenaar',
     'admin' => 'Beheerder',
     'subuser' => 'Medegebruiker',
+    'author' => 'Schrijver',
+    'image_uri' => 'Image URI',
```

Comment:
Translation missing

Reply:
Image is Docker image, cannot translate that to Afbeelding.

URI is also URI, afaik

Reply:
Image is Docker image, cannot translate that to Afbeelding.

URI is also URI, afaik